queries/java/highlights.scm

```diff
@@ -59,32 +73,35 @@
   scope: (identifier) @type)
  (#match? @type "^[A-Z]"))
 
-(constructor_declaration
-  name: (identifier) @type)
-
-(type_identifier) @type
-(boolean_type) @type.builtin
-(integral_type) @type.builtin
-(floating_point_type) @type.builtin
-(floating_point_type) @type.builtin
-(void_type) @type.builtin
+[
+(boolean_type)
+(integral_type)
+(floating_point_type)
+(floating_point_type)
+(void_type)
+] @type.builtin
 
 ; Variables
 
 ((identifier) @constant
  (#match? @constant "^_*[A-Z][A-Z\d_]+"))
 
 (identifier) @variable
-
 (this) @variable.builtin
```

Comment:
@vigoux we have no hightlight for this yet. In python, we use `@constant.builtin`

Reply:
No`@variable.bultin`? I thought we had... We will need to fix that later...

Reply:
This needs to be at least commented out with a `TODO`...

Reply:
done (TODO)